src/Plugin/Field/FieldFormatter/StrawberryPagedFormatter.php

```diff
@@ -423,16 +423,13 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
       // A rendered Manifest
       $manifest = '';
       switch ($pagestrategy) {
-        case 'json_key':
-
-          break;
+	case 'json_key':
+		break;
```

Comment:
Tiny tiny change (sorry) could you indent it with a only two spaces instead of 4 (2 spaces = 1 tab)? It is not a concern really but once i have to do DCS it will probably complain. I can merge anyway, but if you manage to do it before even better. Thanks so much!

Reply:
Sorry, I checked all indents are 2 spaces, I hope it's ok now.